combat-trainer.lic

```diff
@@ -104,8 +106,28 @@ class SetupProcess
 
     game_state.wield_whirlwind_offhand
   end
-
+  
+  def toggle_ah(mode)
```

Comment:
💡 'toggle' in the name here threw me off since this function does not toggle (i.e. flip), rather it sets the var to a given value. Do you think another name could be clearer?